ibis/backends/postgres/registry.py

```diff
@@ -431,6 +431,14 @@ def _array_slice(t, op):
     return sa_arg[sa_start + 1 : sa_stop]
 
 
+def _array_index(t, op):
+    sa_array = t.translate(op.arg)
+    sa_index = t.translate(op.index)
+    if isinstance(op.arg, ops.Literal) and op.arg.dtype.is_array():
```

Comment:
No need to check `is_array` here, since this is true by definition of `op.arg`.

Reply:
You're right. Fixed.